### PR TITLE
Converted Twitter to X

### PR DESCRIPTION
**Pull request by TimGels**
Twitter is no more. We should use X now.

### CODE REVIEW

**Review by felix920506 — APPROVED** [review]
~~What should we call tweets now that twitter is no more~~

**Comment by JPVenson** [comment]
i am generally fine with changing the texts but would argue for sake of readability to keep twitter for code side of things

**Review by JPVenson — CHANGES_REQUESTED** [review]
i am generally fine with changing the texts but would argue for sake of readability to keep twitter for code side of things

**Comment by TimGels** [comment]
> i am generally fine with changing the texts but would argue for sake of readability to keep twitter for code side of things

what about xcorp?

**Comment by TimGels** [comment]
> Yes, I agree with @JPVenson on that. I don't care what Elon wants to call his site, but `button-x` is so generic it's unreadable. Let's leave everything that isn't actually user-facing as-is. Or use `x-formerly-twitter`.

https://en.wikipedia.org/wiki/X_Corp. xcorp seems to be the new legal entity. i like that more than still dragging twitter along

**Comment by JPVenson** [comment]
`button-xcorp` is _fine_ with me. Just not _just_ `x`

File: src/components/contact/XCorpCard.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <h3 className='margin-bottom--none' style={{ flexGrow: 1 }}>
+        X
+      </h3>
+      <SiX />
+    </div>
+    <div className='card__body'>
+      Follow us on X for release announcements and other updates, along with general musings.
+    </div>
+    <div className='card__footer'>
+      <a href='https://www.x.com/jellyfin' className='button button--block button--x'>
```
Comment (JPVenson) [inline src/components/contact/XCorpCard.tsx:18]:
```suggestion
      <a href='https://www.x.com/jellyfin' className='button button--block button--xcorp'>
```

**Review by JPVenson — CHANGES_REQUESTED** [review]

**Review by felix920506 — APPROVED** [review]
LGTM

File: src/components/contact/XCorpCard.tsx (hunk 1 above)
Comment (TimGels) [reply src/components/contact/XCorpCard.tsx:18]: Good catch

**Review by JPVenson — APPROVED** [review]
lgtm

**Review by joshuaboniface — APPROVED** [review]

**Comment by nielsvanvelzen** [comment]
We should just remove it, the Twitter (yes Twitter because look at the domain) account isn't even active anymore. A lot of stuff we posted on other social channels (even our Telegram channel!) was never posted to Twitter.

Also, you've changed the meta tags to invalid ones.

**Pull request merged**